src/LanguageServer/Impl/Sources/DefinitionSource.cs

```diff
@@ -38,13 +39,13 @@ internal sealed class DefinitionSource {
             }
 
             var eval = analysis.ExpressionEvaluator;
-            using (eval.OpenScope(analysis.Document, exprScope)) {
+            var name = (expr as NameExpression)?.Name;
```

Comment:
Can we find anything at all if `expr` is not a `NameExpression`?

Reply:
Member expression also works.